### PR TITLE
Add rust example using Axum framework and Tokio runtime

### PR DESCRIPTION
**Pull request by erikjuhani**
The output is constructed using html macro, which generates Virtual Nodes and replicates "jsx" – style. Output is also compressed into gzip to match with other rendering comparisons.

Result cases are intentionally ignored due to the nature of this comparison example. We can always assume that we end in the happy path, and if not it's ok to panic.

The crate has all optimizations enabled in dev profile (`opt-level = 3`).

The players data is slurped into memory of the application. The virtual nodes are created on-demand when a request to root path `/` is received. The app runs on port 8080.

Add ignore cases directly related to the rust-example like `rust-axum/target` and `rust-axum/assets`.

### CODE REVIEW

File: rust-axum/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+1. Install rust 
+
+```sh
+curl --proto '=https' --tlsv1.2 https://sh.rustup.rs -sSf | sh
+```
+
+2. Build tailwind styles
+
+```sh
+npx tailwindcss -i ./tailwind.css -o ./assets/styles.css`
```
Comment (mkotamies) [inline rust-axum/README.md]: End of of line has extra ` :)

File: rust-axum/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+```
+
+3. Start the server
+
+```sh
+cargo run .
```
Comment (mkotamies) [inline rust-axum/README.md:32]: Should the cargo be installed separately? I did rust default install but looks like I don't have cargo installed

**Comment by mkotamies** [comment]
Looks great and I wouldn't have considered rust myself. Thanks!

File: rust-axum/README.md (hunk 1 above)
Comment (erikjuhani) [reply rust-axum/README.md]: Haha good catch 😄 Fixed.

**Pull request merged**